app/controllers/stories.controller.js

```diff
@@ -0,0 +1,20 @@
+const { startCohereChat } = require('../services/llm-service');
+
+exports.create = async (req, res) => {
+  try {
+    const userId = req.params.userId;
+    const storyParams = req.body.storyParams;
+
+    // Call the LLM service to generate the story text
+    const storyText = await startCohereChat(storyParams);
+
+    //TODO - save storyText to dB here before sending in the response.
+    
+    res.send(storyText);
+  } catch (error) {
+    console.error("Error creating story:", error);
```

Comment:
helpful tip for an app that may be used by many people at the same time, include as much context in the "logged" error as possible, e.g.,
```javascript
console.error(`When creating story for user ${userId} with params ${storyParams} encountered error ${error}`);
```

there's also, potentially, two different places that could cause an error here, the call to `startCohereChat` and the call to `send` the story text as part of the result, maybe those two things should be in separate `try..catch` blocks so you can distinguish in your error log where things went wrong?

Reply:
Updated with more descriptive logging. Thanks for the feedback.